src/components/ActivityButtons/ActivityButtons.js

```diff
@@ -1,25 +1,83 @@
-import React from 'react'
+import React, { useState } from 'react'
 import classes from './ActivityButtons.module.scss'
 import PropTypes from 'prop-types'
+import { DateTime } from 'luxon'
+
+const ActivityButtons = ({ activity, link, suggestActivity, startOver }) => {
+const [ saved, setSaved ] = useState(false)
+  const saveActivity = () => {
+    const activityToSave = {
+      date: DateTime.local().toLocaleString(),
+      description: activity,
+      link: link || null,
+    }
+    addToLocalStorage(activityToSave)
+  }
+  
+  const addToLocalStorage = (activityToSave) => {
+    if (localStorage.getItem('savedActivities') === null) {
+      let activities = [activityToSave]
+      localStorage.setItem('savedActivities', JSON.stringify(activities))
+      setSaved(true)
+    } else {
+      const activities = JSON.parse(localStorage.getItem('savedActivities'))
+      activities.push(activityToSave)
+      localStorage.setItem('savedActivities', JSON.stringify(activities))
+      setSaved(true)
+    }
+  }
```

Comment:
Very nice, just like the old days. But seriously, it's good to remember how to do this.